app/forms/reservation_form.rb

```diff
@@ -9,6 +9,15 @@ class ReservationForm < Reform::Form
   property :license, on: :work_version, default: 'none'
   property :title, on: :work_version
 
+  def deserialize!(params)
+    params['license'] = collection.required_license if collection.required_license
```

Comment:
#1979 was about a default license.  Is required_license what we ought to use here?

Reply:
Oh, I didn't realize there was a difference, but now I see there is. Let me check with @amyehodge.

Reply:
@justinlittman I see you made a change here.  Did you talk to @amyehodge about it?

Reply:
Done.